src/DynamoInstall/Definitions.wxs

```diff
@@ -7,19 +7,19 @@
     </Fragment>
```

Comment:
This file does not need to be included in commit (since it's generated when installer is built), only `Product.wxs` is important. You can verify this by deleting this file, and build the installer again, there shouldn't be any problem. Once that's confirm, please push another commit here so it's removed from your PR. Thanks!